README.md

```diff
@@ -13,7 +13,7 @@ subsystem (aka SocketCAN):
 * canplayer : replay CAN logfiles
 * cansend : send a single frame
 * cangen : generate (random) CAN traffic
-* cansniffer : display CAN data content differences (just 11bit CAN IDs)
+* cansniffer : display CAN data content differences (~~just 11bit CAN IDs~~ Not any more!)
```

Comment:
Just remove the comment in the about the 11 bit. We've git to have a history of the source code.